charts/steward/data/pipelineruns-default-networkpolicy.yaml

```
@@ -21,7 +21,7 @@ spec:
         - 10.0.0.0/8 # address block for private networks
         - 100.64.0.0/10 # shared address space for carrier-grade NAT
         - 169.254.0.0/16 # address block for local links; used for metadata.google.internal
-        - 172.16.0.0/12 # address block for private networks; used for K8s masters
+        - 172.16.0.0/12 # address block for private networks; used for K8s <span class="x x-first x-last">control plane</span>
```

```suggestion
        - 172.16.0.0/12 # address block for private networks; used for K8s instance
```
Be consistent with test/networkpolicy/open_policy_aws.yaml